src/diffusers/pipelines/stable_diffusion/convert_from_ckpt.py

```diff
@@ -1046,30 +1134,28 @@ def load_pipeline_from_original_stable_diffusion_ckpt(
     if "state_dict" in checkpoint:
         checkpoint = checkpoint["state_dict"]
 
-    with tempfile.TemporaryDirectory() as tmpdir:
-        if original_config_file is None:
-            key_name = "model.diffusion_model.input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight"
-
-            original_config_file = os.path.join(tmpdir, "inference.yaml")
-            if key_name in checkpoint and checkpoint[key_name].shape[-1] == 1024:
-                if not os.path.isfile("v2-inference-v.yaml"):
-                    # model_type = "v2"
-                    r = requests.get(
-                        " https://raw.githubusercontent.com/Stability-AI/stablediffusion/main/configs/stable-diffusion/v2-inference-v.yaml"
-                    )
-                    open(original_config_file, "wb").write(r.content)
-
+    if original_config_file is None:
+        key_name = "model.diffusion_model.input_blocks.2.1.transformer_blocks.0.attn2.to_k.weight"
+        if key_name in checkpoint and checkpoint[key_name].shape[-1] == 1024:
+            if not controlnet:
+                config_file_name = "v2-inference-v.yaml"
                 if global_step == 110000:
                     # v2.1 needs to upcast attention
                     upcast_attention = True
             else:
-                if not os.path.isfile("v1-inference.yaml"):
-                    # model_type = "v1"
-                    r = requests.get(
-                        " https://raw.githubusercontent.com/CompVis/stable-diffusion/main/configs/stable-diffusion/v1-inference.yaml"
-                    )
-                    open(original_config_file, "wb").write(r.content)
+                config_file_name = "cldm_v21.yaml"
+        else:
+            if not controlnet:
+                config_file_name = "v1-inference.yaml"
+            else:
+                config_file_name = "cldm_v15.yaml"
 
+        original_config_file = hf_hub_download(
+            'lint/stable_diffusion_configs',
```

Comment:
we shouldn't use a personal hub repo

Reply:
Got it, I've added a comment note to change the repo link in the final commit. If its a good idea to host it on hub, lmk what repo link to put in